City-of-Helsinki/kuvaselaamo
# HEL-237 | Show better error if image_id is invalid

Previously if user entered "Image view" with incorrect `image_id`, error 500 page was shown. Now user should see image with text "Image you were searching was not found"

Testing instructions:
1) Enter search image view
2) Select random image
3) Change `image_id` to something random
4) Expect to see image with text "Image you were searching was not found"
